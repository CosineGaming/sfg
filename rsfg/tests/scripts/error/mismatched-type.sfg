fn main()
	one(5)

fn one(x: str)

