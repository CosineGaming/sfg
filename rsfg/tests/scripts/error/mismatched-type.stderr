[ERROR] mismatched type, expected str at 4:8, got int at 2:6